docs/topics/data-analysis/data-analysis-visualization.md

```diff
@@ -1,23 +1,175 @@
+> 
+{type="note"}
 
 ## Create a line chart
 
-Some basic examples?
+Let's create a line chart with the `df` data frame in the Kotlin Notebook.
+
+Use the `plot()` function from the Kandy library. Within the `plot()` function, specify the type of chart (in this case, it's `line`) 
+and the values for the X and Y axes. You can customize colors and sizes:
+
+```kotlin
+df.plot {
+    line {
+        // Accesses the data frame's columns used for the X and Y axes 
+        x(Month)
+        y(Temperature)
+        // Accesses the data frame's column used for categories and sets colors for these categories 
+        color(City) {
+            scale = categorical("Berlin" to Color.PURPLE, "Madrid" to Color.ORANGE, "Caracas" to Color.GREEN)
+        }
+        // Customizes the line's size
+        width = 1.5
+    }
+    // Customizes the chart's layout size
+    layout.size = 1000 to 450
+}
+```
+
+Here's the result:
+
+![Line chart](visualization-line-chart.svg){width=600}
+
+## Create a points chart
+
+Now, let's visualize the `df` data frame in a points (scatter) chart. 
+
+Within the `plot()` function, specify the `points` chart type. Add the X and Y axes' values and the categorical values from the `df` columns.
+You can also include a heading to your chart:
+
+```kotlin
+df.plot {
+    points {
+        // Accesses the data frame's columns used for the X and Y axes 
+        x(Month) { axis.name = "Month" }
+        y(Temperature) { axis.name = "Temperature" }
+        // Customizes the point's size
+        size = 5.5
+        // Accesses the data frame's column used for categories and sets colors for these categories 
+        color(City) {
+            scale = categorical("Berlin" to Color.LIGHT_GREEN, "Madrid" to Color.BLACK, "Caracas" to Color.YELLOW)
+        }
+    }
+    // Adds a chart heading
+    layout.title = "Temperature per month"
+}
+```
+
+Here's the result:
+
+![Points chart](visualization-points-chart.svg){width=600}
 
 ## Create a bar chart
 
-## Create an area chart
+Finally, let's create a bar chart grouped by city using the same data as in the previous charts. 
+For colors, you can also use hexadecimal codes: 
+
+```kotlin
+// Groups by cities  
+df.groupBy { City }.plot {
+    // Adds a chart heading
+    layout.title = "Temperature per month"
+    bars {
+        // Accesses the data frame's columns used for the X and Y axes 
+        x(Month)
+        y(Temperature)
+        // Accesses the data frame's column used for categories and sets colors for these categories 
+        fillColor(City) {
+            scale = categorical(
+                "Berlin" to Color.hex("#6F4E37"),
+                "Madrid" to Color.hex("#C2D4AB"),
+                "Caracas" to Color.hex("#B5651D")
+            )
+        }
+    }
+}
+```
+
+Here's the result:
+
+![Bar chart](visualization-bar-chart.svg){width=600}
+
+<p>&nbsp;</p>
```

Comment:
This is to add extra space besides the one that is set automatically. I tested with and without extra space. Without extra space, the note is so close to the graph that it is information related to that specific section, while it's information applicable to all the sections. This is how I solved it previously with notes like that: https://kotlinlang.org/docs/wasm-troubleshooting.html#firefox

Would you happen to have any recommendation to avoid the note appearing as part of the previous section?

Reply:
I don't think that double space helps to underline that this info is for all the sections

I thinks this note fits better in the Create the data frame section, that where readers actually use data and see records in columns. other sections are about visualization and different plot types

You can replace the current green note (with 2 links) with this one. And move these links to the What's next section

Reply:
That's a great idea. Changed it.